### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by vhrychenko**
- Developer(s): @vhrychenko

- Ticket: https://spryker.atlassian.net/browse/ECO-3533

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Inxmail               | minor                 |                       |

-----------------------------------------

##### Change log

Improvements

- Fixed dependencies.
- Added metafiles.
- Fixed transfers.
- Fixed bridges.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,29 +1,52 @@
 {
   "name": "spryker-eco/inxmail",
+  "type": "library",
   "description": "Inxmail module",
+  "license": "proprietary",
```
Comment (dereuromark) [inline composer.json]: License file says MIT

File: composer.json [hunk 2]
```diff
@@ -1,29 +1,52 @@
...
+    "spryker/customer-extension": "^1.0.0",
     "spryker/guzzle": "^2.2.0",
-    "spryker/oms": "^8.0.0 || ^10.0.0 || ^11.0.0",
-    "spryker/customer": "^7.10.0",
-    "spryker/mail": "^4.0.0"
+    "spryker/kernel": "^3.0.0",
+    "spryker/locale": "^1.0.0 || ^2.0.0 || ^3.0.0",
+    "spryker/money": "^1.0.0 || ^2.0.0",
+    "spryker/product": "^5.0.0 || ^6.0.0",
+    "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0",
```
Comment (dereuromark) [inline composer.json:15]: This is kinda crazy..
Are we sure this is true? 
Did you manually download 5.0.0 and replaced the core here just to have a quick smoke test?

File: composer.json [hunk 3]
```diff
@@ -1,29 +1,52 @@
...
     "spryker/code-sniffer": "*",
-    "spryker/monolog": "*",
-    "spryker/queue": "*",
-    "spryker/testify": "*"
+    "spryker/mail": "*",
+    "spryker/oms": "*"
+  },
+  "suggest": {
+    "spryker/mail": "If you want to use mail plugins.",
+    "spryker/oms": "If you want to use oms plugins."
```
Comment (dereuromark) [inline composer.json]: OMS (acronym)

File: composer.json (hunk 2 above)
Comment (vhrychenko) [reply composer.json:15]: There is only one usage of SalesFacade via a bridge. I think it's true. The problem that the first version of module was released without correct dependencies. I don't want to use the last versions of the dependencies to not force customer projects to update other modules.

Comment (vhrychenko) [reply composer.json:15]: I think it should work for all the versions correctly.

/**
     * @param int $idSalesOrder
     *
     * @return \Generated\Shared\Transfer\OrderTransfer
     */
    public function getOrderByIdSalesOrder($idSalesOrder);

File: composer.json (hunk 3 above)
Comment (alex-galych) [reply composer.json]: Fixed.

**Review by alex-galych — APPROVED** [review]

**Pull request merged**